### ISSUE TITLE
Doubly quoted strings work different from Ruby Sass

### ISSUE THREAD

**Issue by ericam** [sass/libsass#1215]
The output from my webfont toolkit changed with the latest libsass update. What used to output `../fonts/rockingham-bolditalic-webfont.woff` now returns `../fonts/"rockingham-bolditalic-webfont.woff"`. I'm having trouble tracking down the source of that problem, but I have found some related string-quoting inconsistencies. 

```scss
.test {
  -quotes: 'this-string' == 'this-string';
  -quotes: this-string == 'this-string';
  -quotes: 'this-string' == "this-string";
  -quotes: 'this-string' == '"this-string"';
  -quotes: '"this-string"' == "'this-string'";
}
```

Libsass returns `true` for all 5 comparisons. Ruby Sass returns `false` for the last 2 — which would seem more appropriate, since they are treated differently by both libsass and ruby sass:

```scss
// input
.inspect {
  -string: 'this-' + '"string"';
  -string: 'this-' + 'string';
}

// output
.inspect {
  -string: 'this-"string"';
  -string: "this-string";
}
```

**Comment by ericam** [comment]
Also, using `@debug`, it is impossible to tell the difference between any of these. The quotes never appear.

**Comment by ericam** [comment]
Oddly, my larger issue was fixed by changing `url('#{$font-path}#{$path}')` to `url($font-path + $path)`. Does that make sense to anyone?

**Comment by xzyfer** [comment]
Spec added https://github.com/sass/sass-spec/pull/397

**Comment by xzyfer** [comment]
Apologies for the radio silence @ericam. This will be fixed in 3.2.5.
